buddyboss/buddyboss-platform
# PROD-724 #3039 - Fix: Editing a post with a tagged user is generating a link preview

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3039 .

### How to test the changes in this Pull Request:

1. Go to the Activity feed
2. Post and tag a user
3. Edit post and can't see the preview 

### Proof Screenshots or Video
https://vimeo.com/629876496/c77db1585f


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
